modules/social_features/social_group/social_group.install

```diff
@@ -696,3 +699,78 @@ function social_group_update_10302() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Import flag "Mute Group Notifications" configurations.
+ */
+function social_group_update_10303() {
+  $configs = [
+    'install' => [
+      'flag.flag.mute_group_notifications',
+    ],
+  ];
+
+  ConfigurationsService::importConfigurations($configs, 'social_group');
+}
+
+/**
+ * Update the permissions for social_group.
+ */
+function social_group_update_10304() {
+  $permissions = [
+    'flag mute_group_notifications',
+    'unflag mute_group_notifications',
+  ];
+
+  $roles = Role::loadMultiple();
+
+  /** @var \Drupal\user\Entity\Role $role */
+  foreach ($roles as $role) {
+    if ($role->id() === 'anonymous') {
+      continue;
+    }
+
+    user_role_grant_permissions($role->id(), $permissions);
+  }
+}
+
+/**
+ * Update User Groups view configurations.
+ */
+function social_group_update_10305() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_group', 'social_group_update_10305');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Import exposed block filter configurations of User Groups view.
+ */
+function social_group_update_10306() {
+  $configs = [
+    'optional' => [
```

Comment:
Also counts for `config/optional` which is also not a static representation, it can change over time.
Imagine users coming from 10.0.0 and going to 11.0.0 in the future, the content of these yml files might have changed and could create issues with dependencies etc.

Reply:
I've changed it as well. Check it, please.